mkt/abuse/serializers.py

```diff
@@ -52,3 +54,13 @@ class WebsiteAbuseSerializer(BaseAbuseSerializer):
 
     class Meta(BaseAbuseSerializer.Meta):
         fields = BaseAbuseSerializer.Meta.fields + ('website',)
+
+
+class ExtensionAbuseSerializer(BaseAbuseSerializer):
+    extension = SplitField(
+        SlugOrPrimaryKeyRelatedField(source='extension', slug_field='slug',
+                                     queryset=Extension.objects.all()),
```

Comment:
My point was that you shouldn't be able to report non-public websites, hence my queryset suggestion. So I don't think using the slug is going to be an issue.